app_dart/lib/src/service/cache_service.dart

```diff
@@ -5,28 +5,85 @@
 import 'dart:convert';
 import 'dart:typed_data';
 
+import 'package:meta/meta.dart';
 import 'package:neat_cache/cache_provider.dart';
 import 'package:neat_cache/neat_cache.dart';
 
-import '../datastore/cocoon_config.dart';
-
 class CacheService {
-  CacheService(this.config);
-
-  final Config config;
-
-  CacheProvider<List<int>> _provider;
-
-  Future<Cache<Uint8List>> redisCache() async {
-    _provider = Cache.redisCacheProvider(await config.redisUrl);
-    return Cache<List<int>>(_provider)
-        .withCodec<Uint8List>(const _CacheCodec());
+  CacheService({
+    bool inMemory = false,
+    int inMemoryMaxSize = 256,
```

Comment:
What's the unit? 256 Kb or Mb? It's generally a good idea to add unit to the name, e.g. `inMemoryMaxSizeMb`.

Reply:
I renamed it to `inMemoryMaxNumberEntries` and added class documentation further explaining it.